File: GXpert/GXpert.Web/Modules/Content/ContentAudio/ContentAudioForm.cs
using Serenity.ComponentModel;
using System;

namespace GXpert.Content.Forms;

[FormScript("Content.ContentAudio")]
[BasedOnRow(typeof(ContentAudioRow), CheckNames = true)]
public class ContentAudioForm
{
    public string MediaFile { get; set; }
    public string HlsFile { get; set; }
    public string FileKeyUrl { get; set; }
    public string Iv { get; set; }
    public int ContentId { get; set; }
   // public int LanguageId { get; set; }
    public int UserId { get; set; }
   
}

File: GXpert/GXpert.Web/Modules/ServerTypes/Content/ContentAudioForm.ts
import { StringEditor, LookupEditor, PrefixedContext, initFormType } from "@serenity-is/corelib";

export interface ContentAudioForm {
    MediaFile: StringEditor;
    HlsFile: StringEditor;
    FileKeyUrl: StringEditor;
    Iv: StringEditor;
    ContentId: LookupEditor;
    LanguageId: LookupEditor;
    UserId: LookupEditor;
}

export class ContentAudioForm extends PrefixedContext {
    static readonly formKey = 'Content.ContentAudio';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!ContentAudioForm.init)  {
            ContentAudioForm.init = true;

            var w0 = StringEditor;
            var w1 = LookupEditor;

            initFormType(ContentAudioForm, [
                'MediaFile', w0,
                'HlsFile', w0,
                'FileKeyUrl', w0,
                'Iv', w0,
                'ContentId', w1,
                'LanguageId', w1,
                'UserId', w1
            ]);
        }
    }
}